src/api/app/controllers/trigger_controller.rb

```diff
@@ -12,6 +12,11 @@ class TriggerController < ApplicationController
 
   def runservice
     auth = request.env['HTTP_AUTHORIZATION']
+
+    if request.env['HTTP_X_GITLAB_EVENT'] = 'Push Hook'
```

Comment:
I think you forgot an equal sign here in the `if`

Reply:
yep, fixed